bobtimus/src/loan.rs

```diff
@@ -352,6 +358,12 @@ fn validate_loan_is_acceptable(
         }));
     }
 
+    if terms.iter().find(|a| a.days == request_term).is_none() {
```

Comment:
Clippy is not complaining? 

You could also `.any` here (but not that important): 


```
    if !terms.iter().any(|a| a.days == request_term) {
        return Ok(Err(LoanValidationError::TermNotAllowed {
            term: request_term,
        }));
    }
```